.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 jobs:
   publish:
     docker:
-      - image: circleci/node:erbium
+      - image: cimg/node:12.22.9
```

Comment:
We don't have any local version files here but similar to https://github.com/PagerDuty/pdjs/pull/317#discussion_r789082219 maybe we can update this to Node 16.13?

Reply:
👍